modules/javafx.controls/src/main/java/javafx/scene/control/skin/TableColumnHeader.java

```diff
@@ -758,11 +761,12 @@ private <T,S> void resizeColumnToFitContent(TreeTableView<T> ttv, TreeTableColum
                 double w = cell.prefWidth(-1);
 
                 maxWidth = Math.max(maxWidth, w);
-                tableSkin.getChildren().remove(cell);
             }
         }
+        tableSkin.getChildren().remove(treeTableRow);
 
```

Comment:
would we need

treeTableRow.updateTreeTableView(null);

here to prevent a memory leak?

Reply:
I don't think so. The row and cell should be gc'ed after this method since there are not used anywhere